CorePayments/src/main/java/com/paypal/android/corepayments/api/CoreRequestFactory.kt

```diff
@@ -0,0 +1,13 @@
+package com.paypal.android.corepayments.api
+
+import com.paypal.android.corepayments.APIRequest
+import com.paypal.android.corepayments.HttpMethod
+import com.paypal.android.corepayments.api.models.GetOrderRequest
+
+open class CoreRequestFactory {
+
+    fun createGetOrderInfoRequest(getOrderRequest: GetOrderRequest): APIRequest {
```

Comment:
Looks like the get order additions are still in this PR, when we don't need them anymore